src/none.js

```diff
@@ -0,0 +1,28 @@
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * Returns `true` if no elements of the list match the predicate,
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a -> Boolean) -> [a] -> Boolean
+ * @param {Function} fn The predicate function.
+ * @param {Array} list The array to consider.
+ * @return {Boolean} `true` if the predicate is not satisfied by every element, `false` otherwise.
+ * @example
+ *
+ *      R.none(isNaN)([1, 2, 3, NaN]); //=> false
+ *      R.none(isNaN)([1, 2, 3]); //=> true
+ */
+module.exports = _curry2(function(fn, list) {
+    var idx = -1;
+    while (++idx < list.length) {
```

Comment:
@davidchambers :+1:  np! I've changed it already. However, I don't quite understand why: I was under the impression that accessing the length of an Array had a complexity of O(1), so I don't see how this will make it different. Would you mind explaining what I'm missing here? Thanks!